server/migrations/1661465391816_allocation_migration.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-restricted-syntax, no-await-in-loop */
+const { dbClient, collections } = require('../db');
+
+exports.shorthands = 'migrate-phase-data';
+
+exports.up = async () => {
+  const queries = [
+    // Setup initial phase
+    `INSERT INTO ${collections.GLOBAL_PHASE} (
+            "name", start_date, end_date, created_by, updated_by, created_at, updated_at
+        ) values(
+            'Initial Phase', '2020-01-01', '2022-12-31', 'system', 'system', now(), now())`,
+
+    // Copy site's allocation to initial phase
+    `INSERT INTO ${collections.SITE_PHASE_ALLOCATION} 
+    (phase_id, site_id, allocation, start_date, end_date, created_by, updated_by, created_at, updated_at)
+        SELECT
+            1,
+            id,
+            CAST(COALESCE(body ->> 'allocation', '0') AS integer),
+            NULL,
+            NULL,
+            'system',
+            'system',
+            now(),
+            now()
+        FROM
+            employer_sites`,
+    // Verification: Error will be raised if migration failed and transaction will fail
+    `DO $$
```

Comment:
This block just verifies that the numbers are correct and allocations are the same, yeah?

Reply:
That's right!